src/Juvix/Compiler/Asm/Translation/FromTree.hs

```diff
@@ -227,6 +227,7 @@ genCode fi =
       Tree.PrimUnop op' -> mkUnop op'
       Tree.OpTrace -> mkInstr Trace
       Tree.OpFail -> mkInstr Failure
+      Tree.OpAnomaGet -> impossible
```

Comment:
Should this impossible be an unsupported error instead?

Reply:
This translation does not throw errors so I don't know how I could achieve this.

Reply:
solved in 40be37daaaebbfbb65108c8e0093542c44f4e735